helm/sematic-server/templates/configmap.yaml

```diff
@@ -15,6 +15,10 @@ data:
 {{ if .Values.aws.enabled }}
   AWS_S3_BUCKET: {{ .Values.aws.storage_bucket | quote }}
   STORAGE: "sematic.plugins.storage.s3_storage.S3Storage"
+{{ end }}
+{{ if .Values.slack.enabled }}
+  SLACK_WEBHOOK_TOKEN: {{ .Values.slack.slack_webhook_token | quote }}
+  SLACK_EXTERNAL_URL: {{ .Values.slack.slack_external_url | quote }}
```

Comment:
Nit: `SLACK_EXTERNAL_URL` makes it sound like this should be a Slack URL. How about we name this one of:
- `SEMATIC_EXTERNAL_URL`
- `SEMATIC_URL`

Reply:
I think we need a general server setting, not one tied to Slack specifically. There may be other times when we need to allow other systems to post to the Sematic server, and it would be a pain to have to configure the same URL to the same thing N times. It could be called `SEMATIC_EXTERNAL_URL`, as Emmanuel suggests.

Reply:
Yes, this.
But `SEMATIC_DASHBOARD_URL`.